addons/web/static/src/xml/notification.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+<t t-name="Notification">
+    <div t-attf-class="o_notification #{widget.className}" t-translation="off" role="alertdialog">
```

Comment:
Should use bootstrap toast component, the notification style has to adapt with the theme